microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md

```diff
@@ -26,26 +26,53 @@ The IE compatibility List only adds a website when the following actions occur.
-- gives the user the option to explore Microsoft Edge
+On redirection to Microsoft Edge, the user is shown the one-time dialog in the next screenshot.  The dialog provides the user with the following information.  
+
+*   It explains why the website is being redirected.  
+*   It prompts the user for consent to copy browsing data and preferences from IE to Microsoft Edge.  
+
+:::row:::
+   :::column span="":::
+      The following browsing data is imported.  
+      
+      *   Favorites  
+      *   Passwords  
+      *   Search engines  
+      *   Open tabs  
+      *   History  
+      *   Settings  
+      *   Cookies  
+      *   The Home Page  
+   :::column-end:::
+   :::column span="":::
+      :::image type="complex" source="../media/neededge-dialog1.msft.png" alt-text="Browsing notification and prompt to import data and preferences" lightbox="../media/neededge-dialog1.msft.png":::
+         Browsing notification and prompt to import data and preferences  
+      :::image-end:::  
+   :::column-end:::
+:::row-end:::
+
+If the user does not consent by choosing the **Always bring over my browsing data and preferences from Internet Explorer** checkbox, the user may choose **Continue browsing** to continue the browsing session.  
+
+Finally, a website incompatibility banner is displayed under the address bar for each redirection.  An example of a website incompatibility banner is displayed in following figure.
+
+:::image type="complex" source="../media/neededge-banner.msft.png" alt-text="Notification about modern sites and prompt to set Microsoft Edge as default browser or explore Microsoft Edge" lightbox="../media/neededge-banner.msft.png":::
+   Notification about modern sites and prompt to set Microsoft Edge as default browser or explore Microsoft Edge  
+:::image-end:::
+
+The website incompatibility banner provides the following details to the user.  
+
+*   Recommends that the user to switch to Microsoft Edge.  
+*   Offers to set Microsoft Edge as the default browser.  
+*   Gives the user the option to explore Microsoft Edge.    
+    
+When a website is redirected from Internet Explorer to Microsoft Edge, one of the following actions occurs.
 
-When a site is redirected from Internet Explorer to Microsoft Edge, the Internet Explorer tab that started loading the site is closed if it had no prior content. Otherwise, the active tab view goes to a  Microsoft [support](https://support.microsoft.com/office/the-website-you-were-trying-to-reach-doesn-t-work-with-internet-explorer-8f5fc675-cd47-414c-9535-12821ddfc554?ui=en-US&rs=en-US&ad=US) page that explains why the site was redirected to Microsoft Edge.
+*   If the active IE tab had no prior content, it is closed.  
+*   If the active IE tab had prior content, it navigates to the [Microsoft support page that explains why the website was redirected to Microsoft Edge][MicrosoftSupportOfficeTheWebsiteYouWereTryingToReachDoesntWorkWithInternetExplorer].  
 
 > [!NOTE]
 > After 
```

Comment:
This is missing the following sentence. Can you please update 

"After a redirection users can go back to using Internet Explorer for sites that are not on the Internet Explorer incompatibility list."

Reply:
Updated.